local-cli/rnpm/windows/index.js

```diff
@@ -1,33 +1,6 @@
+const minimistConfig = require('./src/config');
```

Comment:
>minimistConfig [](start = 6, length = 14)

This actually uses commander, not minimist AFAIK

Reply:
Will fix